<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadGameDialog</class>
 <widget class="QDialog" name="LoadGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Game</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select a PDN file to load:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="fileNameLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>No file selected</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="fileListView"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
</ui>